src/server/src/main/java/io/cassandrareaper/service/MetricsGrabber.java

```diff
@@ -41,6 +44,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(MetricsGrabber.class);
 
   private final AppContext context;
+  private final ExecutorService executor = Executors.newFixedThreadPool(5);
```

Comment:
This needs to instead be a dropwizard lifecycle based executor, and also preferably wrapped with metrics instrumentation.

see work done in https://github.com/thelastpickle/cassandra-reaper/commit/b4c32482e600699c41e90ba8c9f517b81a870bf6

Reply:
or if it is not used in the class it should just be removed.